src/fieri/app/models/foodcritic_worker.rb

```diff
@@ -8,21 +8,32 @@ def perform(params)
     begin
       cookbook = CookbookArtifact.new(params['cookbook_artifact_url'], jid)
     rescue Zlib::GzipFile::Error => e
-      logger = Logger.new File.new(File.expand_path('./log/fieri.log'))
-      logger.error e
-    else
-      feedback, status = cookbook.criticize
+      format_log_message(e)
+    end
 
-      Net::HTTP.post_form(
-        URI.parse("#{ENV['FIERI_SUPERMARKET_ENDPOINT']}/api/v1/cookbook-versions/foodcritic_evaluation"),
-        fieri_key: ENV['FIERI_KEY'],
-        cookbook_name: params['cookbook_name'],
-        cookbook_version: params['cookbook_version'],
-        foodcritic_feedback: feedback,
-        foodcritic_failure: status
-      )
+    feedback, status = cookbook.criticize
 
-      cookbook.cleanup
+    begin
+      make_post(params, feedback, status)
+    rescue
+      format_log_message(e)
     end
+    cookbook.cleanup
+  end
+
+  def format_log_message(e)
+    logger = Logger.new File.new(File.expand_path(ENV['FIERI_LOG_PATH']))
```

Comment:
Have you considered using [the logger built in to a Sidekiq worker](https://github.com/mperham/sidekiq/wiki/Logging) already? Our Sidekiq service logs to STDOUT, which in our omnibus install is managed by runit which in turn writes out STDOUT in production to `/var/log/supermarket/sidekiq/current`.